core/src/main/java/com/netflix/conductor/core/utils/JsonUtils.java

```diff
@@ -0,0 +1,68 @@
+package com.netflix.conductor.core.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.Map;
+
+@Singleton
+public class JsonUtils {
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public JsonUtils() {
+    }
+
+    private Object getJson(String jsonAsString) {
+        try {
+            return objectMapper.readValue(jsonAsString, Object.class);
+        } catch (Exception e) {
+            return jsonAsString;
+        }
+    }
+
+    @SuppressWarnings("unchecked")
```

Comment:
Consider adding javadoc to this method since it can call private methods depending on the instance type.

Reply:
public method does not need `@VisibleForTesting`

Reply:
Added javadoc and cleaned up.